case_study1_airline_safety.Rmd

```diff
@@ -0,0 +1,106 @@
+# Case study #1 : Airline Safety data
+
+    ```{r setup, include=FALSE}
+    knitr::opts_chunk$set(include = FALSE)
+    library(ggplot2)
+    library(ggpubr)
+    library(haven)
+    library(dplyr)
+    ```
+
+## Setup
+
+This case study is based on an [article](https://fivethirtyeight.com/features/should-travelers-avoid-flying-airlines-that-have-had-crashes-in-the-past/) from [FiveThirtyEight](https://fivethirtyeight.com/) published by [Nate Silver](https://fivethirtyeight.com/contributors/nate-silver/). Using [data](https://github.com/fivethirtyeight/data/tree/master/airline-safety) from the [Aviation Safety Network’s database](https://aviation-safety.net/index.php), we want to see if past safety records can predict future risk of accidents. In other words "Should Travelers Avoid Flying Airlines That Have Had Crashes in the Past?"
+
+    ```{r include = FALSE}
+    ## need to find the best way to let students download the data 
+    data=read.csv2("~/Documents/TA_Econometrics/Case_studies/airline_safety/data-master/airline-safety/airline-safety_long.csv", sep = ";")
+    ```
```

Comment:
yes:
```
read.csv(file = "https://raw.githubusercontent.com/fivethirtyeight/data/master/airline-safety/airline-safety.csv")
```

Reply:
the options I see are : 
- dropbox 
- github but I fear that a lot of them cannot manage to download from this 
- send it by email 
- moodle 

may be the best is moodle, right ?

Reply:
ok. just it into `inst/datasets`.